From 901836f4a798eb81d28a208f9195ea1bc9316ee0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Wed, 7 Dec 2011 09:31:13 +0800
Subject: [PATCH] Revert "earlysuspend: Temporary enable debug"

This reverts commit 9b6c7f264c2d38c93d5acac94ce9760894611a71.

Disable earlysuspend debug for release.
---
 kernel/power/earlysuspend.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/kernel/power/earlysuspend.c b/kernel/power/earlysuspend.c
index 86e14ef1898c..b15f02eba45c 100644
--- a/kernel/power/earlysuspend.c
+++ b/kernel/power/earlysuspend.c
@@ -28,7 +28,7 @@ enum {
 	DEBUG_SUSPEND = 1U << 2,
 	DEBUG_VERBOSE = 1U << 3,
 };
-static int debug_mask = DEBUG_USER_STATE | DEBUG_SUSPEND | DEBUG_VERBOSE;
+static int debug_mask = DEBUG_USER_STATE;
 module_param_named(debug_mask, debug_mask, int, S_IRUGO | S_IWUSR | S_IWGRP);
 
 static DEFINE_MUTEX(early_suspend_lock);
@@ -107,8 +107,6 @@ static void early_suspend(struct work_struct *work)
 		pr_info("early_suspend: sync\n");
 
 	sys_sync();
-	if (debug_mask & DEBUG_SUSPEND)
-		pr_info("early_suspend: done\n");
 abort:
 	spin_lock_irqsave(&state_lock, irqflags);
 	if (state == SUSPEND_REQUESTED_AND_SUSPENDED)
-- 
2.35.3

